### ISSUE TITLE
Add `isNull` function in `io.neow3j.devpack.Helper`

### ISSUE THREAD

**Issue by hurui200320** [neow3j/neow3j#530]
Hi,

Today I'm trying to figure out how to know if the given key has already in the `StorageMap`. I tried `.length != 0`:

```java
if (someMap.get(key).length != 0){
    // ...
}
```

And it gives me a `Invalid type for SIZE: Any` which I guess it means `NullPointerException` in Java, since if key not in the map, there is no data to fetch.

Then I tried `!= null`:

```java
if (someMap.get(key) != null){
    // ...
}
```

Which gives me `Specified cast is not valid.`. I'm not sure what that means but by using `throw new Exception();` (I'm using IDEA so I can't use the debug tool), I figured out the problem does is the `!= null`, I guess it tries to convert the byte[] to something and compared with NULL type.

Then I found out there is a NeoVM opcode called `ISNULL`, which test the top element and return true if is null and false for not null.
Thus I wrote a function to use that:
```Java
    @Instruction(opcode = OpCode.ISNULL)
    public static native boolean isNull(Object any);
```
And it works fine:
```java
if (!isNull(someMap.get(key))) {
    // ...
}
```

I think this would be a useful function and I'll be glad to have it in the Helper class.

**Comment by csmuller** [comment]
Hey @hurui200320, thanks for the detailed description. You already seemed to have understood the devpack well, by writing your own method with a `@Instruction` annotation 👏 💪 

I will run a couple of tests here with the null check, because it seems to me that there is a bug.
I will reply to you soon again.

**Comment by csmuller** [comment]
The problem that we are facing here is as follows:
The neow3j compiler relies on `JMPIFNOT` to check if a value is either false or null. The documentaiton on that neo-vm opcode says that the program will jump if the value on the stack is null. But, if we take a closer look in the vm we can see that, in case of a ByteString stack item this only works if the item is not longer than 32 bytes. In your example the value retrieved form the storage is longer than 32 bytes.

Solution, adapt the compilation such that the ISNULL opcode is used explicitly for null checks.

**Comment by csmuller** [comment]
The issue has been solved in #532. It will be released soon in version 3.9.0.
